server/src/test/resources/csars/yaml/valid/lamp-csar/template-no-input.yaml

```diff
@@ -0,0 +1,81 @@
+tosca_definitions_version: tosca_simple_yaml_1_0
```

Comment:
Why do we need a template without inputs?
If you decide to keep it, make sure to put it in its own folder (own scripts, etc), and adjust the scripts.
You don't pass inputs to the scripts, so I don't see how this would work at all.

Reply:
I thought a template without inputs might be useful for early testing since I'm not sure how we're going to implement inputs.

As for the multiple folder question, they used to be in two folders, but they were merged into one, I'm trying to figure out why though.

Reply:
I also think that an template without inputs would be good for testing purposes. Maybe we could completely separate both templates into different folders